server/config/brain/chat/questions.en.json

```diff
@@ -1,12 +1,71 @@
 [
   {
     "label": "chat.greeting",
-    "questions": ["Hey!"],
-    "answers": ["Hey!"]
+    "questions": ["Hey!", "Hi", "Hello"],
+    "answers": ["Hey!", "Hi", "Hello, I'm Gladys."]
+  },
+  {
+    "label": "chat.greeting-gladys",
+    "questions": ["Hey Gladys !", "Hi Gladys", "Hello Gladys"],
+    "answers": [
+      "Hey, I'm your home automation assistant !",
+      "Hello, I am your home automation assistant !",
+      "Hi, I am your home automation assistant !"
+    ]
   },
   {
     "label": "chat.i-love-you",
     "questions": ["I love you"],
-    "answers": ["Me too!", "Of course you love me..."]
+    "answers": ["Me too!", "I knew that ..."]
+  },
+  {
+    "label": "chat.joke",
+    "questions": ["Gladys tell me a joke.", "Gladys do you know a joke ?", "Tell me a joke."],
+    "answers": [
+      "Do you know the devil's joke? Too bad it's hell !",
+      "The other day I told a joke about my clothes. They were folded !",
+      "Who's in charge of the decoration at the Elysee? The Home Office.",
+      "What do you say when calling a 4-headed monster? Hello, hello, hello, hello !",
+      "What do a postman and a knife thrower have in common? Both must have a lot of skill.",
```

Comment:
Does it work in english? I don't see the joke.

Reply:
I don't think so, as I don't see it either.